openid-caep-interoperability-profile-1_0.md

```diff
@@ -175,6 +193,28 @@ Receivers MUST be prepared to accept events with any of the subject identifier f
 ## Event Signatures
 All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
 
+## OAuth Service
+
+### Authorization Server
+* SHALL distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [RFC8414]{{RFC8414}}
+* MUST support at least one of the following to obtain a short-lived access token 
+** client credential grant flow
+** authorization code flow 
+
+### OAuth Scopes
+
+* Authorization server MUST reserve the scopes for the SSF endpoints with the prefix of `ssf`
```

Comment:
"Authorization server" -> "An OAuth {{RFC6749}} authorization that is used to issue tokens to SSF Receivers"...